deploy/charts/cert-manager/templates/service.yaml

```diff
@@ -4,6 +4,10 @@ kind: Service
 metadata:
   name: {{ template "cert-manager.fullname" . }}
   namespace: {{ .Release.Namespace | quote }}
+{{- with .Values.serviceAnnotations }}
```

Comment:
I followed the value naming pattern used with `.Values.serviceLabels` below on line 17, but I wonder if it might be more appropriate to use `.Values.prometheus.serviceAnnotations` instead. Let me know what your thoughts are on this.

Reply:
For consistency with the labels value naming `serviceAnnotations` make sense to me 👍🏼